server/api_routes/userRoutes.js

```diff
@@ -35,9 +36,14 @@ router.post('/', (req, res, next) => {
           error: 'Password must be between 8 to 24 characters long!',
         });
       } else {
-        return User.create(req.body).then(newUser => {
+        return User.create({ email, password }).then(async (newUser) => {
```

Comment:
Need to include firstName and lastName in user creation. They are set to allowNull: false. So maybe, 
```
const {firstName, lastName, email, password} = req.body
const createUserObj = {firstName, lastName, email, password}

Then, 
return User.create(createUserObj)
```

Reply:
good catch! corrected.